Assets/_BForBoss/_Character/FPS_Camera_Blends.asset

```diff
@@ -35,47 +35,3 @@ MonoBehaviour:
         m_PreInfinity: 0
         m_PostInfinity: 0
         m_RotationOrder: 0
-  - m_From: CM Crouch
```

Comment:
Is this change related to Shapes?

Reply:
This was a side fix. Removed all the blends related to 3rd person.